train.py

```diff
@@ -172,6 +172,14 @@ def train(index,args):
                             ).per_device_loader(device)
                 score = evaluate(model=model, dataloader=val_dataloader, tokenizer=tokenizer, gpt_tokenizer=gpt_tokenizer, wandb=wandb)
                 wandb.log({"val_"+i:j for i,j in score.items()})
+                
+                with open("score") as f:
```

Comment:
@SunCreation
minor, how about adding an extension `txt`
```suggestion
                with open("score.txt") as f:
```

Reply:
Thank you for checking :)